build/tasks/styles.js

```diff
@@ -127,7 +123,7 @@ export default async function compileStyles(sassOptions = null, postcssOptions =
             infile  = resolve(infile);
             outfile = resolve(outfile);
 
-            let result = await sassRender(Object.assign({}, sassOptions, {
+            let result = sass.compile(infile, Object.assign({}, sassOptions, {
                 file: infile,
                 outFile: outfile,
             }));
```

Comment:
According to [Sass' documentation of the `compile` function](https://sass-lang.com/documentation/js-api/functions/compile/), the `file` and `outFile` options don't exist (might be logic from an older version of this script).

```suggestion
            let result = sass.compile(infile, Object.assign({}, sassOptions));
```